inst/include/temperature_component.hpp

```diff
@@ -72,27 +72,27 @@ class TemperatureComponent : public IModelComponent {
     void setoutputs(int tstep);
 
     // Hard-coded DOECLIM parameters
-    const int dt = 1;                     // years per timestep (this is implicit in Hector)
+    double dt = 1;                     // years per timestep (this is implicit in Hector)
     int ns;                               // number of timesteps
-    const double ak = 0.31;               // slope in climate feedback - land-sea heat exchange linear relationship
-    const double bk = 1.59;               // offset in climate feedback - land-sea heat exchange linear relationship, W/m2/K
-    const double csw = 0.13;              // specific heat capacity of seawater W*yr/m3/K
-    const double earth_area = 5100656E8;  // m2
-    const double kcon = 3155.0;           // conversion from cm2/s to m2/yr
-    const double q2co = 3.7;              // radiative forcing for atmospheric CO2 doubling, W/m2
-    const double rlam = 1.43;             // factor between land clim. sens. and sea surface clim. sens. T_L2x = rlam*T_S2x
-    const double secs_per_Year = 31556926.0;
-    const double zbot = 4000.0;           // bottom depth of diffusive ocean, m
-    const double bsi = 1.3;               // warming factor for marine surface air over SST (due to retreating sea ice)
-    const double cal = 0.52;               // heat capacity of land-troposphere system, W*yr/m2/K
-    const double cas = 7.80;              // heat capacity of mixed layer-troposphere system, W*yr/m2/K
-    const double flnd = 0.29;             // fractional land area
-    const double fso = 0.95;              // ocean fractional area below 60m
+    const double ak = 0.31;               // slope in climate feedback - land-sea heat exchange linear relationship (W/(m2K))
+    const double bk = 1.59;               // offset in climate feedback - land-sea heat exchange linear relationship,(W/(m2K))
+    const double csw = 0.13;              // specific heat capacity of seawater (Wyr/(m3K))
+    const double earth_area = 5100656E8;  // (m2)
+    const double secs_per_Year = 60.0 * 60.0 * 24.0 * 365.2422; //  secs. * min. * hrs. * tropical calendar days (seconds)
```

Comment:
In the ocean component 

https://github.com/JGCRI/hector/blob/master/src/ocean_component.cpp#L203
`double time = 60*60*24*365.25;  // seconds per year`

we use a different number of seconds per year different by 11 minutes... do we care that they are different? should they be consistent?

Reply:
Probably we don't care enough  🤷‍♂️